src/collector.rs

```diff
@@ -13,7 +13,10 @@
 /// handle.pin().flush();
 /// ```
 
-use std::sync::Arc;
+#[cfg(feature = "nightly")]
```

Comment:
Maybe the `cfg_if` crate would be useful for implementing this branch.  @stjepang used it in https://github.com/crossbeam-rs/crossbeam-epoch/pull/39/files#diff-7f3d2612bf5d28e2e8a5055f7ecfdc8fR32